src/interpreter/mod.rs

```diff
@@ -243,8 +243,11 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
 
             Str(ref s) => {
                 let ptr = self.memory.allocate(s.len(), 1)?;
-                self.memory.write_bytes(ptr, s.as_bytes())?;
-                self.memory.freeze(ptr.alloc_id)?;
+                // don't write to the zst allocation
```

Comment:
@oli-obk Can it just be ignored in `get_mut`? I still don't get what the problem is. This code worked before. What's changed?